templates/slurm.conf.j2

```diff
@@ -137,6 +137,14 @@ PartitionName={{part.name}} \
         {% endfor %}{# group #}
 {% endfor %}{# partitions #}
 
+
+# Define slurmd nodes not in partitions for configless login-only nodes:
+{% for node in hostvars %}
+    {% if hostvars[node].openhpc_login_only %}
```

Comment:
Wonder if we could supply a list of nodes not to put in partition?

```
ohpc_slurmd_no_partition_nodes: "{{ groups['login] }}"
```

Would this be any better? Might be better if limit didn't do an incomplete templating of slurm.conf (although I think this is an issue already)?